lib/rocket_pants/test_helper.rb

```diff
@@ -95,16 +92,26 @@ def process(action, *args)
 
       # Rails 4 changes the method signature. In rails 3, parameters is the first argument.
       # In Rails 4, it's the second.
-      if args.first.is_a?(String)
-        parameters = (args[1] ||= {})
+      if Rails::VERSION::MAJOR <= 4
+        if args.first.is_a?(String)
+          parameters = (args[1] ||= {})
+        else
+          parameters = (args[0] ||= {})
+        end
       else
         parameters = (args[0] ||= {})
       end
 
       response.recycle_cached_body!
 
-      if _default_version.present? && parameters[:version].blank? && parameters['version'].blank?
-        parameters[:version] = _default_version
+      if Rails::VERSION::MAJOR <= 4
```

Comment:
this whole test_helper.rb is awful and just garbage. it's trying to prop up test controller implementation details in order to mock a controller.